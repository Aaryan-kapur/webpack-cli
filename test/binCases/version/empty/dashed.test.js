'use strict';

const { run } = require('../../../testUtils');
const serializer = require('jest-serializer-ansi');

test('dashed', () => {
    const { stdout, stderr } = run(__dirname, ['--version']);
    expect.addSnapshotSerializer(serializer);
    expect(stdout).toMatchSnapshot();
    expect(stderr).toHaveLength(0);
});
