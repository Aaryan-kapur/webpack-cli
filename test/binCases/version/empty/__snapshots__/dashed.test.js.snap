// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dashed 1`] = `
"
webpack-cli 4.0.0-alpha-5
webpack 4.35.3
"
`;
